### DISCUSSION TITLE
TypeScript: code formatting: not the same as `eslint --fix`

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#4262] [Q&A] [answered]
What formatter do you use? Is it possible to use `eslint --fix`?

**Comment by chemzqm** [comment, marked as answer]
You may need coc-eslint, check the readme.

**Reply by vitaly-zdanevich** [reply]
coc-eslint is not mentioned in README.md

**Reply by chemzqm** [reply]
Readme of https://github.com/neoclide/coc-eslint
